src/app/ubs-admin/components/ubs-admin-employee/employee-form/employee-form.component.ts

```diff
@@ -171,6 +174,9 @@ export class EmployeeFormComponent implements OnInit {
   private transferFile(imageFile: File): void {
     this.isWarning = this.showWarning(imageFile);
 
+    this.imageIsTooLarge = imageFile.size > this.maxImageSize;
+    this.invalidImageFormat = imageFile.type !== 'image/jpeg' && imageFile.type !== 'image/png';
```

Comment:
It would be good to create a separate array with all valid file types like
`private vaildImageFormats = ['image/jpeg', 'image/png'];`
and then you can rewrite that part to
`this.invalidImageFormat = !this.vaildImageFormats.includes(imageFile.type);`
It's more elegant way and it will be easier to add more supported types in future

Reply:
Done